### PR TITLE
{lang}[GCCcore/11.3.0] Python v3.10.4

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

notes:
* I removed `nose` as it failed to install. This is from 2015 and no longer maintained.
* I removed `paycheck` as it failed to install. This is from 2011 and no longer maintained.
* package versions that are not latest:
  * `packaging` 20.9: `poetry 1.1.13 has requirement packaging<21.0,>=20.4`
  * `docutils` 0.17.1: `Sphinx 4.5.0 has requirement docutils<0.18,>=0.14`
  * `platformdirs` 2.4.1: `platformdirs >= 2.5 requires hatchling and that requires packaging>=21.3`

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/db5ad1bb4f81a24552630dd6c89af403 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+    }),
+    ('pyasn1', '0.4.8', {
+        'checksums': ['aef77c9fb94a3ac588e87841208bdec464471d9871bd5050a287cc9a475cd0ba'],
+    }),
+    ('PyNaCl', '1.5.0', {
+        'modulename': 'nacl',
+        'checksums': ['8ac7448f09ab85811607bdd21ec2464495ac8b7c66d146bf545b0f08fb9220ba'],
+    }),
+    ('bcrypt', '3.2.0', {
+        'checksums': ['5b93c1726e50a93a033c36e5ca7fdcd29a5c7395af50a6892f5d9e7c6cfbfb29'],
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb:152]:
```suggestion
    ('bcrypt', '3.2.2', {
        'checksums': ['433c410c2177057705da2a9f2cd01dd157493b2a7ac14c8593a16b3dab6b6bfb'],
```

File: easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+        'checksums': ['cf98038d2abf63a1ada5730e91e84c642ba6c225b0198c3684151b1f80c5f8a6'],
+    }),
+    ('webencodings', '0.5.1', {
+        'checksums': ['b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923'],
+    }),
+    ('html5lib', '1.1', {
+        'checksums': ['b2e5b40261e20f354d198eae92afc10d750afb487ed5e50f9c4eaf07c184146f'],
+    }),
+    ('cleo', '0.8.1', {
+        'source_tmpl': '%(name)s-%(version)s-py2.py3-none-any.whl',
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb:443]:
```suggestion
        'source_tmpl': 'SOURCE_WHL',
```

**Review by SebastianAchilles — COMMENTED** [review]
Do we want to update `bcrypt` to latest?

I think for `cleo` we can better use the template `SOURCE_WHL`.

File: easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb]: They've released two new versions (3.2.1 and 3.2.2) since I did this yesterday!

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a8e7f241f1d8063f30e9a486be7ecf9f for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/ec9c2925d8c5ea6bb4402f05b3707e78 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/d7a7e5edf18e885e898cada556def067 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/74a22626a2fed0bb099bcf99cc7d6f5d for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2722.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/5a5c0a1cb0a7345fd7a008688d14e78b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
fair-mastodon-c6g-2xlarge-0001 - Linux Rocky Linux 8.5, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/bf8a24fecc8eae6fe0c1b566afad1835 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3134.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d90935d2f615e58b1e562d73c4fa6168 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node112.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/643996f4d6f5f33fe7bb817fcf2cdf98 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**